src/ZoneServer/Buffs/Handlers/Scout/DoubleAttack_Buff.cs

```diff
@@ -1,7 +1,7 @@
 ﻿using Melia.Shared.Game.Const;
 using Melia.Zone.Buffs.Base;
 
-namespace Melia.Zone.Buffs.Handlers
+namespace Melia.Zone.Buffs.Handlers.Scout
```

Comment:
We can discuss this more, but I feel like it's inevitable that we need some kind of sorting because there's literally hundreds of buffs.  If we ever have to move a buff, it's at least quite simple to do, due to the way the handlers work skills applying the buff don't need to care about its namespace, so moving it from one folder to another is as simple as a drag and drop.

Reply:
There's actually _thousands_ of buffs, but yea, I don't disagree^^